tests/apps/sequencing_metrics_parser/parsers/test_bcl2fastq_parser.py

```diff
@@ -0,0 +1,89 @@
+from unittest import mock
+from cg.apps.sequencing_metrics_parser import (
+    get_yield_values,
+    get_read_metrics,
+    get_index_metrics,
+    get_flow_cell_name,
+    get_sample_id,
+    get_number_of_reads_for_sample_in_lane,
+    get_lane_number,
+    get_yield_q30,
+    get_lane_yield_in_bases,
+    get_total_clusters_passing_filter,
+    get_total_raw_clusters,
+    get_quality_score,
+    get_lane_yield_q30_values,
+    get_lane_read_quality_score_values,
+    get_perfect_reads_for_sample_in_lane,
+    parse_bcl2fastq_sequencing_metrics,
+)
+from cg.apps.sequencing_metrics_parser.models import SequencingMetricsForLaneAndSample
+
+
+def test_get_flow_cell_name(sequencing_metrics_json, flow_cell_name):
+    assert get_flow_cell_name(sequencing_metrics_json) == flow_cell_name
```

Comment:
Type hints in the args and a brief docstring would be nice in all the functions in this file